packages/lib/src/index.ts

```diff
@@ -5,10 +5,11 @@ if (process.env.NODE_ENV === 'development') {
     // require('preact/debug');
 }
 
+import {CoreOptions} from 'src/core/types';
 import Checkout from './core';
 /* eslint-enable */
 
-async function AdyenCheckout(props) {
+async function AdyenCheckout(props: CoreOptions) {
```

Comment:
Could you also add the return type as well?

```suggestion
async function AdyenCheckout(props: CoreOptions): Promise<Checkout> {
```

Reply:
Totally! Added it via 75ea617

Reply:
Great, thanks!